src/validateSchema.js

```diff
@@ -8,20 +8,30 @@ const getValidationErrors = (schema, model) => Object.keys(schema).reduce((acc,
   const value = model[key]
   const rules = schema[key]
 
+  const renderError = (condition, fallback) => {
+    let err;
+    try {
+      err = rules.formatError({ key, value, condition, rules, schema, model });
+    } catch (e) {
+      err = rules.formatError || fallback;
```

Comment:
Thank you for the explanation, that all makes sense. Ultimately, I side with preferring explicit type checks over using `try/catch` for control flow like this, especially because there's an assumption being made about the value's type type when the process throws. It may give the user a little more leeway with their usage, but it doesn't keep the API as tight.

Luckily, if you want a string, it's as easy as doing `formatError: () => 'my error message'`, and this has the added benefit of keeping the API clean and consistent.

It's a pretty minor piece in the grand scheme of things, but I do like to be extra cautious :). I'll submit a PR to your branch and, if it looks good to you, we can then get your PR merged and published. Thanks for taking the time to contribute!